### ISSUE TITLE
DeprecationWarning: executable_path has been deprecated

### ISSUE THREAD

**Issue by satishkumar96** [SergeyPirogov/webdriver_manager#262]
**Body of my code is:** 

```
import pytest
from selenium import webdriver
from webdriver_manager.chrome import ChromeDriverManager
from webdriver_manager.firefox import GeckoDriverManager


@pytest.fixture(params=["chrome"])
def init_driver(request):
    if request.param == "chrome":
        driver = webdriver.Chrome(ChromeDriverManager().install())
    if request.param == "firefox":
        driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())
    request.cls.driver = driver
    driver.implicitly_wait(30)
    driver.set_page_load_timeout(30)
    driver.maximize_window()
    yield
    driver.quit()
```
I am using Python 3.10.0 and PyCharm Version: 2021.2.3 (Community edition) , after I run the above code it display warning message in terminal as:

> DeprecationWarning: executable_path has been deprecated, please pass in a Service object
>     driver = webdriver.Chrome(ChromeDriverManager().install())
> 
> -- Docs: https://docs.pytest.org/en/stable/warnings.html

Can you please solve this issue?

**Comment by aleksandr-kotlyar** [comment]
@satishkumar96 hi!
I have discovered that this warning is from Selenium4 version.
It's not the webdriver-manager problem.
Just use the code for your line with firefox driver:
```python
from selenium import webdriver
from selenium.webdriver.firefox.service import Service
from webdriver_manager.firefox import GeckoDriverManager

driver = webdriver.Firefox(service=Service(executable_path=GeckoDriverManager().install()))
```
And the warning should disappear.

**Comment by satishkumar96** [comment]
> @satishkumar96 hi!
> 
> ## Root cause
> This warning is from Selenium4 version.
> 
> ## Conclusion
> It's not the webdriver-manager problem.
> 
> ## Solution
> Just use the code for your line with firefox driver:
> 
> ```python
> from selenium import webdriver
> from selenium.webdriver.firefox.service import Service
> from webdriver_manager.firefox import GeckoDriverManager
> 
> driver = webdriver.Firefox(service=Service(executable_path=GeckoDriverManager().install()))
> ```
> 
> And the warning should disappear. Cheers!

Thanks @aleksandr-kotlyar the code is running smoothly without an error or warnings.

**Comment by Sn3akyP3t3** [comment]
Surely to run into this error not described above:
ValueError: API Rate limit exceeded. You have to add GH_TOKEN

Solution: https://github.com/SergeyPirogov/webdriver_manager#gh_token

**Comment by aleksandr-kotlyar** [comment]
@Sn3akyP3t3 thats not a point of this issue. The problem with gh token has come in 3.6.x-3.7.0 releases but it was fixed in 3.7.1
